applications/bwa_alignment/values.yaml

```diff
@@ -0,0 +1,109 @@
+values:
+  - name: ReferenceGenomeURL
+    display_name: Reference Genome URL
+    uint_value:
+    string_value: https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/005/845/GCF_000005845.2_ASM584v2/GCF_000005845.2_ASM584v2_genomic.fna.gz
+
+  - name: Read1URL
+    display_name: Read 1 URL
+    uint_value:
+    string_value: https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/NA12878/Garvan_NA12878_HG001_HiSeq_Exome/NIST7035_TAAGGCGA_L001_R1_001.fastq.gz
+
+  - name: Read2URL
+    display_name: Read 2 URL
+    uint_value:
+    string_value: https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/NA12878/Garvan_NA12878_HG001_HiSeq_Exome/NIST7035_TAAGGCGA_L001_R2_001.fastq.gz
+
+  - name: ReferenceGenomeName
+    display_name: Reference Genome Name
+    uint_value:
+    string_value: GCF_000005845.2_ASM584v2
+
+  - name: BwaContainerImage
+    display_name: BWA Container Image
+    uint_value:
+    string_value: docker://biocontainers/bwa:v0.7.17_cv1
+
+  - name: GithubUploadImage
+    display_name: GitHub Upload Image
+    uint_value:
+    string_value: docker://alpine:latest
+
+  - name: BwaIndexCpuCores
+    display_name: BWA Index CPU Cores
+    uint_value: 4
+    string_value:
+
+  - name: BwaIndexMemory
+    display_name: BWA Index Memory
+    uint_value: 8
+    string_value:
+
+  - name: BwaAlignCpuCores
+    display_name: BWA Align CPU Cores
+    uint_value: 16
+    string_value:
+
+  - name: BwaAlignMemory
+    display_name: BWA Align Memory
+    uint_value: 32
+    string_value:
+
+  - name: BwaAlignThreads
+    display_name: BWA Align Threads
+    uint_value: 16
+    string_value:
+
+  - name: GithubUploadCpuCores
+    display_name: GitHub Upload CPU Cores
+    uint_value: 1
+    string_value:
+
+  - name: GithubUploadMemory
+    display_name: GitHub Upload Memory
+    uint_value: 1
+    string_value:
+
+  - name: GithubToken
+    display_name: GitHub Token
```

Comment:
maybe make that a secret?

Reply:
Or just do away with it altogether in favor of supplying an optional S3 URL and secret.